alacritty.yml

```diff
@@ -592,8 +592,8 @@
 # - Vi mode exclusive actions:
 #
 #   - Open
-#       Open URLs at the cursor location with the launcher configured in
-#       `url.launcher`.
+#       Perform the action of the first matching hint, with `mouse.enabled`,
+#       under the cursor.
```

Comment:
You've changed everything minor I didn't mind much and skipped the thing I was actually concerned about. :smile: 

Also I'm not sure just stating `with mouse.enabled` is sufficient, since technically that could be any of the two values. I'd maybe suggest the following:

```suggestion
#       Perform the action of the first matching hint under the vi mode cursor with `mouse.enabled` set to `true`.
```

Feel free to make suggestions yourself though please.

Reply:
Oh god damn it... I had added it in one rewording, then managed to delete it in my frantic search for a way to write this.

Reply:
> Also I'm not sure just stating with `mouse.enabled` is sufficient, since technically that could be any of the two values.

I'm quite confident anyone can manage to understand that this implies a value of `true`, frankly.

Reply:
I'm mostly just trying to be cautious, since we're using up two lines already anyways. With yaml there always has to be a value associated, so just saying `mouse.enabled` has to be there technically doesn't document what that value is supposed to be.